cli/create.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+/**
+ * @memberof CLI
+ * @desc
+ * Calls to repository to query for Objects are done through this sub-command
+ */
+
+//
+// Imports
+//
+var log = require("../lib/util/log");
+var cmdCore = require("./core");
+var fs = require("fs");
+var path = require("path");
+//
+// Action
+//
+module.exports = function(file, options) {
+    cmdCore.init(options);
+
+    var filePath = path.resolve(file);
+
+    fs.stat(filePath, function(err, stat) {
+        cmdCore.handleError(err);
+
+        if (stat.isFile()) {
+
+            var filestr = fs.readFileSync(filePath, 'utf8');
+            var data;
+
+            try {
+                data = JSON.parse(filestr);
+            }
+            catch(err) {
+                cmdCore.handleError(err);
+            }
+						log.debug(data);
```

Comment:
Ugh shucks sorry. Editor was in boomerang mode :(
Am 15.10.2015 18:07 schrieb "Matt Solnit" <notifications@github.com>:

> In cli/create.js
> <https://github.com/SOASTA/repository.js/pull/26#discussion_r42144415>:
>
> > +
> > +    fs.stat(filePath, function(err, stat) {
> > +        cmdCore.handleError(err);
> > +
> > +        if (stat.isFile()) {
> > +
> > +            var filestr = fs.readFileSync(filePath, 'utf8');
> > +            var data;
> > +
> > +            try {
> > +                data = JSON.parse(filestr);
> > +            }
> > +            catch(err) {
> > +                cmdCore.handleError(err);
> > +            }
> > +						log.debug(data);
>
> Some weird whitespace on this line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SOASTA/repository.js/pull/26/files#r42144415>.
>